src/pages/spawn/spawnVolume/spawnVolumeTable/spawnVolumeTableActions/MigrateBtn.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import Button, { Size } from "@leafygreen-ui/button";
+import { MyVolume } from "types/spawn";
+import { MigrateVolumeModal } from "./migrateBtn/MigrateVolumeModal";
+
+interface Props {
+  volume: MyVolume;
+}
+
+export const MigrateBtn: React.VFC<Props> = ({ volume }) => {
+  const [openModal, setOpenModal] = useState(false);
+
+  return (
+    <>
+      <Button
```

Comment:
This button should be disabled while the volume is migrating
<img width="1180" alt="image" src="https://user-images.githubusercontent.com/9298431/199546979-7aefbaaf-01cf-4bda-bd54-7b59a755047a.png">